index.html

```diff
@@ -0,0 +1,131 @@
+        
+        <div id="logo">
+          <img src="product-landing-page-logo.png" alt="Original Trombones">
+        </div>
+
+        <ul class="nav justify-content-end">
+            <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#features">Features</a>
+            </li>
+
+            <li class="nav-item">
+                <a class="nav-link" href="#how-it-works">How It Works</a>
+            </li>
+
+            <li class="nav-item">
+                <a class="nav-link" href="#pricing">Pricing</a>
+            </li>
+
+        </ul>
+    </header>
+    
+    <section id="hero">
+        <h2>Handcrafted, home-made masterpieces</h2>
+        <form action="https://www.freecodecamp.com/email-submit">
+            <input name="email" id="email" type="email" placeholder="Enter your email address" required>
+            <input id="submit" type="submit" value="GET STARTED" class="btn">
+        </form>
+    </section>
+    
+    <div class="container">
+        <section id="features">
+            <div class="grid">
+                <div class="icon"><i></i></div>
+                <div class="desc">
+                    <h2>Premium Materials</h2>
+                    <p> Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+                </div>
+            </div>
+
+            <div class="grid">
+                <div class="icon"><i></i></div>
+                <div class="desc">
+                    <h2>Fast Shipping</h2>
+                    <p> We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+                </div>
+            </div>
+            
+            <div class="grid">
+                <div class="icon"><i></i></div>
+                <div class="desc">
+                    <h2>Quality Assurance</h2>
+                    <p> For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+                </div>
+            
+                <section id="how-it-works">
+                    <iframe id="video" height="315" src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+                </section>
+               
+                <section id="pricing">
+                    <div class="product" id="tenor">
+                        <div class="level">Tenor Trombone</div>
+                        <h2>$600</h2>
+                        <ol>
+                            <li>Lorem ipsum.</li>
+                            <li>Lorem ipsum.</li>
+                            <li>Lorem, ipsum dolor.</li>
+                            <li>Lorem ipsum.</li>
+                        </ol>
+                        <button class="btn">Select</button>
+                    </div>
```

Comment:
It would be better to not nest a div tag inside the section tag and work only with section tag to make your code more semantic.